examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/adm/ADM.java

```diff
@@ -17,30 +15,46 @@
     public static final SimpleDateFormat ADMTimeFormat = new SimpleDateFormat("HH:mm:ss.SSS");
 
     public static String mkADMConstructor(String constructor, String content) {
-        return constructor + "(\"" + content + "\")";
+        StringBuilder sbConstructor=new StringBuilder();
+        sbConstructor.append(constructor).append("(\"").append(content).append("\")");
+        return sbConstructor.toString();
     }
 
     public static String mkInt64Constructor(long value) {
         return mkADMConstructor("int64", String.valueOf(value));
     }
+
     public static String mkInt8Constructor(String value) {
         return mkADMConstructor("int8", value);
     }
+
     public static String mkInt32Constructor(String value) {
         return mkADMConstructor("int32", value);
     }
+
```

Comment:
Let's not make unnecessary format changes.

Reply:
This function is from Jianfeng, it just add the "int32" prefix to the "String value" without changing the format, the return value is "int32( value )".